pizone/zone.py

```diff
@@ -164,6 +238,6 @@ def _get_zone_state(self, state):
         self._controller._ensure_connected()  # pylint: disable=protected-access  # noqa
         return self._zone_data[state]
 
-    async def _send_zone_command(self, data: Union[str, float]):
-        send_data = {'ZoneNo': str(self._index+1), 'Command': str(data)}
-        await self._controller._send_command_async('ZoneCommand', send_data)
+    async def _send_command(self, command, data: Union[str, float]):
```

Comment:
Why did this function get renamed? Add in an extra function, don't delete the old one please. Rename it ```_send_zone_command_v2``` please.

Reply:
Code re-use. I made the same code applicable to both the ZoneCommand and AirMaxCommand/AirMinCommand. Happy to have a second function that performs a similar function if you wish?

Reply:
Ahh... OK. Refactor ```_send_zone_command``` to have an extra argument then if it's still the v1 API.

Reply:
Are you sure about that naming? ZoneCommand is different to AirMaxCommand/AirMinCommand. I went for the more generic "_send_command" to avoid confusion.

It's just a name so easy change either way.

Reply:
OK then leave it the way you had it.